specification/search/data-plane/Microsoft.Azure.Search.Data/track1/preview/2015-02-28-preview/searchindex.json

```diff
@@ -0,0 +1,297 @@
+{
```

Comment:
We don't need to do this for API versions before 2019-05-06 because the Java SDK won't be supporting those. Sorry I didn't make that clear.

Reply:
Mildly confused - you are saying the "track1" carving out will only have 2019-05-06? (and possibly the next preview version?)

Because this path contains "track1" swagger... the "default" path is "track2"...

Reply:
I forced pushed commits to re-arrange things per what I think you meant @brjohnstmsft . Let me know if that makes sense?